timelock-corruption-detection/src/test/java/com/palantir/history/utils/Utils.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * (c) Copyright 2020 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.history.utils;
+
+import java.nio.ByteBuffer;
+import java.util.UUID;
+import java.util.stream.IntStream;
+
+import com.palantir.paxos.PaxosAcceptorState;
+import com.palantir.paxos.PaxosProposalId;
+import com.palantir.paxos.PaxosStateLog;
+import com.palantir.paxos.PaxosValue;
+
+public final class Utils {
```

Comment:
I'd suggest naming this `PaxosSerializationTestUtils` or something like that, since it's public